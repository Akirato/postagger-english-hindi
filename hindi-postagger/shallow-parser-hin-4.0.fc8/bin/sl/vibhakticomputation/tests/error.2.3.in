9	((	NP	<fs af='raMgaM,n,,sg,,,lo,lo' head="raMgaMlo">
9.1	raMgaMlo	NN	<fs af='raMgaM,n,,sg,,,lo,lo' name="raMgaMlo">
9.2	kUdA	PSP	<fs af='kUdA,avy,,,,,0,0_avy' poslcat="NM">
	))
