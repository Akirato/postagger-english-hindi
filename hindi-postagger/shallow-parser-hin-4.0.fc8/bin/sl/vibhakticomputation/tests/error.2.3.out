1	((	NP	<fs af='raMgaM,n,,sg,,,lo_kUdA,lo' head="raMgaMlo" vpos="vib1_2">
1.1	raMgaMlo	NN	<fs af='raMgaM,n,,sg,,,lo,lo' name="raMgaMlo">
	))		
